bot/pagination.py

```diff
@@ -77,6 +76,7 @@ def add_line(self, line='', *, empty=False):
         RuntimeError
             The line was too big for the current :attr:`max_size`.
```

Comment:
```suggestion
            The line was too big for the current max_size.
```

Reply:
Same as before, prefer to keep wrapped in backticks.